### ISSUE TITLE
Forked with (some) machine-coded data from Euro and Copa

### ISSUE THREAD

**Issue by CorneeldH** [jfjelstul/worldcup#2]
Hi jfjelstul,

Thank you for this repo! Not an issue, more a note: I made a fork and scraped also data from Euro and Copa tournaments. Code and resulting data are available at: https://github.com/CorneeldH/worldcup. If I somehow violated any of your copyright / license please let me know, so I can adapt. 

Again, thanks for sharing the code!

Kind regards, Corneel

**Comment by jfjelstul** [comment]
Cool, I'll take a look. You're welcome to adapt the code as long as you cite this repo (as you do) and release any code or data under the same CC-BY-SA 4.0 license. I plan to publish a dataset on the Euros based on this SQL schema, but I don't have a specific release timeline yet. Let me know if you'd be interested in contributing!

**Comment by CorneeldH** [comment]
Yeah definitely! Of course it depends on the type of contributions you need, but in general I'm happy to help and build :-) For my own planning: I want the coming month(s) to add some other sources (in different repository to keep it more clean) and do some modelling (tidymodels) with it. Except rvest, I'm pretty experienced with R and larger data science projects. All the best, Corneel
